Src/FluentAssertions.Json/JTokenAssertions.cs

```diff
@@ -338,6 +341,37 @@ public AndConstraint<JTokenAssertions> NotMatchRegex(string regularExpression, s
             return new AndWhichConstraint<JTokenAssertions, JToken>(this, jToken);
         }
 
+        /// <summary>
+        ///     Asserts that the current <see cref="JToken" /> has a direct child element with the specified
+        ///     <paramref name="expectedElement" /> name, and the specified <paramref name="expectedValue"/>.
+        /// </summary>
+        /// <param name="expectedElement">The name of the expected child element.</param>
+        /// <param name="expectedValue">The value of the expected child element.</param>
+        /// <param name="because">
+        ///     A formatted phrase as is supported by <see cref="string.Format(string,object[])" /> explaining why the assertion
+        ///     is needed. If the phrase does not start with the word <i>because</i>, it is prepended automatically.
+        /// </param>
+        /// <param name="becauseArgs">
+        ///     Zero or more objects to format using the placeholders in <see paramref="because" />.
+        /// </param>
+        public AndWhichConstraint<JTokenAssertions, JToken> HaveElementWithValue(string expectedElement, string expectedValue, string because = "",
+            params object[] becauseArgs)
+        {
+            if (expectedElement == null)
+                throw new ArgumentNullException(nameof(expectedElement));
+            else if (!expectedElement.Trim().Any())
+                throw new ArgumentException("Invalid expected element name.", nameof(expectedElement));
+
+            JToken jToken = Subject[expectedElement];
+            Execute.Assertion
+                .ForCondition(jToken != null && Subject.Value<string>() == expectedValue)
```

Comment:
What if `expectedValue` is `null`?

Reply:
Then it'll verify that the `JSON` value is `null`.